crates/nu-protocol/src/engine/stack.rs

```diff
@@ -25,7 +26,7 @@ pub type EnvVars = HashMap<String, HashMap<String, Value>>;
 /// in any context. This meant that the parents were going largely unused, with captured variables
 /// taking their place. The end result is this, where we no longer have separate frames, but instead
 /// use the Stack as a way of representing the local and closure-captured state.
-#[derive(Debug, Clone)]
+#[derive(Debug)]
```

Comment:
Also, could you restore the derived `Clone` impl here? The debug assert in the repl should trigger if we try to clone the stack. Thanks!

Reply:
Yep, will do. Thanks for the reminder